test/clj/rems/workflow/test_dynamic.clj

```diff
@@ -190,6 +191,13 @@
                            {:type :application.command/change-resources
                             :actor applicant-user-id
                             :catalogue-item-ids [1 2]}
+                           injections)))
+      (is (= {:errors [{:type :unbundlable-catalogue-items
+                        :catalogue-item-ids [1 2 3]}]}
+             (fail-command application
+                           {:type :application.command/change-resources
+                            :actor applicant-user-id
+                            :catalogue-item-ids [1 2 3]}
```

Comment:
Would it make sense to have separate tests for different form and different workflow? Or is that overkill?